src/ansible_navigator/runner/command_async.py

```diff
@@ -0,0 +1,40 @@
+""" Herewith lies the ability to invoke ansible-runner
+in an async fashion. A queue is provided and ansible-runner
+uses pexpect to parse standout and error from the command run
+and populates the queue with messages
+"""
+
+from queue import Queue
+from ansible_runner import run_command_async
+
+from .command_base import CommandBase
+
+
+class CommandAsync(CommandBase):
+    # pylint: disable=too-few-public-methods
+    # pylint: disable=too-many-instance-attributes
+    """a runner async wrapper"""
+
+    def __init__(self, executable_cmd: str, queue: Queue, **kwargs):
+        """class to handle arguments of ``run_command_async`` interface for ``ansible-runner``.
+           For common arguments refer documentation of ``CommandBaseRunner`` class
+
+        Args:
+            executable_cmd ([str]): The command to be invoked.
+            queue ([Queue]): The queue to post events from ``ansible-runner``
```

Comment:
Look here https://ansible-navigator--623.org.readthedocs.build/en/623/pkg/ansible_navigator.runner.command_async/ — these descriptions are not rendered. Let's use the built-in Sphinx argument description style so that it is.
And also, the types are not really necessary because they are already present in the annotations and there's no need to duplicate/keep in sync manually this info.
```suggestion
        :param executable_cmd: The command to be invoked.
        :param queue: The queue to post events from ``ansible-runner``
```

Reply:
Fixed in #630